.gitignore

```
@@ -9,3 +9,4 @@ yarn.lock
 /node_modules
 /package-lock.json
 /Gemfile.lock
+broken-links.json
```

```suggestion
/.scripts/broken-links.json
```

(and modify the script generating this file as required)